streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -75,6 +89,19 @@ class ReceiverTracker(ssc: StreamingContext, skipReceiverLaunch: Boolean = false
   // This not being null means the tracker has been started and not stopped
   private var endpoint: RpcEndpointRef = null
 
+  private val scheduler: ReceiverScheduler = new LoadBalanceReceiverSchedulerImpl()
+
+  @volatile private var stopping = false
+
+  // Track receivers' status for scheduling
```

Comment:
This is a super-minor style point, but if you write these one-liners as Scaladoc then they'll show up in the quick documentation popups in IntelliJ.  I don't feel super strongly about this and we haven't been consistent about this throughout Spark, but just thought I'd comment since this has been on my mind lately.